.storybook/config.js

```diff
@@ -3,6 +3,7 @@ import { configure } from '@storybook/react';
 function loadStories() {
   require('../stories');
   require('../stories/button');
+  require('../stories/flash_alert');
```

Comment:
Stories (like typical modules that aren't react components) should be `kebab-case`. Only reason we have snake_case in the dashboard is because our project was started by ruby developers 😝